cmd/kops/create_cluster.go

```diff
@@ -147,6 +149,10 @@ func NewCmdCreateCluster(f *util.Factory, out io.Writer) *cobra.Command {
 
 	cmd.Flags().StringVar(&options.MasterSize, "master-size", options.MasterSize, "Set instance size for masters")
 
+	cmd.Flags().StringVar(&options.NodeIAMRole, "node-iam-role", options.NodeIAMRole, "Set IAM role for nodes")
```

Comment:
Hi, @sp-borja-juncosa is on vacations, but I guess that it could be easy to remove the cli flags.